src/Microsoft.DotNet.Interactive.Jupyter.Tests/JupyterKernelTests.cs

```diff
@@ -174,8 +175,10 @@ public async Task can_cancel_submit_code_and_interrupt_kernel()
 
         var sentMessages = options.MessageTracker.SentMessages.ToSubscribedList();
         await waitForCommandReceieved;
-        cts.Cancel();
 
+        await Task.Delay(100);
```

Comment:
Why is this delay needed ?

Reply:
I noticed that this made the test pass locally. I think @colombod was trying an experiment to see if this also makes the test pass in CI.... There is definitely some kind of race here.